README.md

```diff
@@ -172,6 +187,7 @@ go run main.go run \
     --database.password="..." \
     --database.db_name="sidecar"
 ```
+Helpful to use  -p 7100:7100 for testing/development
```

Comment:
Not applicable here since this is not run in docker and port 7100 is the default port